framework/src/multiapps/MultiApp.C

```diff
@@ -590,10 +592,10 @@ MultiApp::restore(bool force)
       }
     }
 
-    _console << "Begining restoring MultiApp " << name() << std::endl;
+    _console << "Restoring MultiApp " << name() << "...";
     for (unsigned int i = 0; i < _my_num_apps; i++)
       _apps[i]->restore(_backups[i]);
-    _console << "Finished restoring MultiApp " << name() << std::endl;
+    _console << "done!" << std::endl;
```

Comment:
Motivation to change this? It seems the old messages are clearer to me

Reply:
This change the output from two lines to one line. It looks a lot nicer I think.

Reply:
`Restored MultiApp ... name() ` is how many lines?

Reply:
Something like this with this change:
```
Restoring MultiApps on TIMESTEP_BEGIN
Restoring MultiApp pebble0...done!
Restoring MultiApp pebble1...done!
Restoring MultiApp pebble2...done!
Restoring MultiApp pebble3...done!
Restoring MultiApp pebble4...done!
Restoring MultiApp pebble5...done!
Restoring MultiApp pebble6...done!
Restoring MultiApp pebble7...done!
Restoring MultiApp pebble8...done!
Finished Restoring MultiApps on TIMESTEP_BEGIN
```